lib/seam/http.rb

```diff
@@ -1,44 +1,51 @@
 # frozen_string_literal: true
 
-require_relative "parse_options"
-require_relative "routes/routes"
+require_relative "http_single_workspace"
 
 module Seam
-  class Http
-    include Seam::Routes
-
-    attr_accessor :defaults
-
-    def initialize(api_key: nil, personal_access_token: nil, workspace_id: nil, endpoint: nil,
-      wait_for_action_attempt: true, debug: false)
-      options = SeamOptions.parse_options(api_key: api_key, personal_access_token: personal_access_token, workspace_id: workspace_id, endpoint: endpoint)
-      @endpoint = options[:endpoint]
-      @auth_headers = options[:auth_headers]
-      @debug = debug
-      @wait_for_action_attempt = wait_for_action_attempt
-      @defaults = Seam::DeepHashAccessor.new({"wait_for_action_attempt" => wait_for_action_attempt})
+  module Http
+    def self.new(**args)
+      Http::SingleWorkspace.new(**args)
     end
 
-    def lts_version
-      Seam::LTS_VERSION
+    def self.from_api_key(api_key, endpoint: nil, wait_for_action_attempt: false, debug: false)
+      Http::SingleWorkspace.from_api_key(api_key, endpoint: endpoint, wait_for_action_attempt: wait_for_action_attempt, debug: debug)
     end
 
-    def request_seam_object(method, path, klass, inner_object, config = {})
-      response = request_seam(method, path, config)
+    def self.from_personal_access_token(personal_access_token, workspace_id, endpoint: nil, wait_for_action_attempt: false, debug: false)
+      Http::SingleWorkspace.from_personal_access_token(personal_access_token, workspace_id, endpoint: endpoint, wait_for_action_attempt: wait_for_action_attempt, debug: debug)
+    end
+
+    class HttpApiError < StandardError
```

Comment:
We don't want to double prefix, e.g., avoid `Http:Http...`
```suggestion
    class ApiError < StandardError
```

Reply:
644f08b05e12b770cb29375192e943de65a7633e